english/be-octodog/octodog.md

```diff
@@ -1,7 +1,7 @@
 This is the secret to become octodog.
 First you have to transformate to an
 octocat by complete this course then
-read at least 100 adventures.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.